tests/serialize/test_simple.py

```diff
@@ -86,3 +86,41 @@ class Error(halogen.Schema):
     )
 
     assert error == {"message": "Ongeldig e-mailadres"}
+
+
+def test_exclude_on_attribute():
+    class FooBar(halogen.Schema):
```

Comment:
a test should test one thing only. Can you split this into multiple tests (either with paramatrize or split manually yourself).